code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -910,9 +911,9 @@ func (fsh *StorageHandler) CreateDir(ctx context.Context, r *http.Request) (*blo
 	formData.ConnectionID = connectionID
 	formData.ActualHash = "-"
 	formData.ActualSize = 1
-
+	// We Need to do a Proper Look here
```

Comment:
@abouroumine What about this comment?

Reply:
The problems revolving around those comments already resolved. just forgot to remove comments.